04-store-api/starter/models/product.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose')
+
+const productSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: [true,'product name must be provided'],
+    }, 
+    price:{
+        type:Number,
+        required:[true, 'product name must be provided'],
```

Comment:
likely typo, should be:

```suggestion
        required:[true, 'product price must be provided'],
```